### ISSUE TITLE
AB3-Collection Type

### ISSUE THREAD

**Issue by edgartdata** [PHAROS-photoarchives/Zeri-Foundation#13]
Should AB3-Collection Type (line 85) be in another section? It doesn't seem to jive with artwork bibliography.

**Comment by marilenadaquino** [comment]
Yes, I do not remember what it actually refers to (photo collection? artwork collection?). It might have ended up there after moving rows around.
